2-0-data-structures-and-algorithms/2-2-6-binary-search-tree/src/test/java/com/bobocode/cs/BinarySearchTreeTest.java

```diff
@@ -260,7 +260,21 @@ void inorderTraversal() {
                  * .........\
                  * ..........5
                  */
-                arguments(new Integer[]{1, 2, 3, 4, 5}, 4));
+                arguments(new Integer[]{1, 2, 3, 4, 5}, 4),
+                /*
+                 * ......10
+                 * ...../..\
+                 * ....6....18
+                 * .../......\
+                 * ..4........20
+                 * ...\........\
+                 * ....5........24
+                 * ..............\
+                 * ...............26
+                 * ................\
+                 * .................30
+                 */
+                arguments(new Integer[]{10,6,18,4,8,12,20,5,24,26,30},5));
```

Comment:
@RepNazar could you please reformat this line to **add missing spaces**?

Reply:
Sorry, forgot to press IDEA shortcut😅